migrations/Version20221024135042.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20221024135042 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'add is_imported to Signalement';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE signalement ADD is_imported BOOLEAN DEFAULT NULL');
+        $this->addSql('UPDATE signalement SET is_imported = 1 WHERE YEAR(created_at) < 2022 AND territory_id != 64 AND territory_id != 59');
```

Comment:
Après vérifications les IDs sont 65 et 60
```
mysql> SELECT id, zip, name FROM `territory` where zip in (59, 64);
```
| id  | zip | name                   |
|------|-----|----------------------|
|  60 | 59  | Nord                   |
|  65 | 64  | Pyrénées-Atlantiques   |
| 102 | 64  | Agglo de Pau           |

3 rows in set, 2 warnings (0.01 sec)

Par contre aucun signalement est rattaché à l'ID 102 donc on peut l'ignorer